Interview and introduction (#68)
WhitakerLab/whitakerlab.github.io: _posts/blog/2019-06-20-Lovely_to_meet_you.md

---
layout: post
title: Lovely to meet you!
categories: blog
image:
  feature:
link:
date: 2019-8-20
modified:
share: true
author: Yini He
---

> So lucky I am to have a great experience studing in the Alan Turing Insititue.

Reviewer: ```suggestion
So lucky I am to have a great experience studying in the Alan Turing Institue.
```

> I feel happy to know many people here,because all of them are zealous,creative and willing to help each other.

Reviewer: ```suggestion
I feel happy to know many people here, and all of them are zealous, creative and willing to help others.
```

> Now,I would like introduce 3 people who I interviewed them.I love their great works.

Reviewer: ```suggestion
Now, I would like to introduce 3 people who I interviewed them. I love their great works.
```

## Ryan Chan

Ryan is a first-year doctoral student at the Alan Turing Institute and Warwick. 

> His smile is full of energy. 

Reviewer: ```suggestion
His smile carries a very powerful energy.
```

He loved and studied mathematics at the University of Leeds, where he developed a keen interest in Bayesian theory and computational statistics. 
His current research interest is broadly concerned with addressing practical constraints of common Monte Carlo algorithms. 
In particular, Ryan’s research focuses on Monte Carlo Fusion, which aims to tackle the problem of unifying distributed analyses and inferences from multiple sources on shared parameters, into a single coherent inference. 

> Applications include expert elicitation, differential privacy and inference in big data settings. 

Reviewer: ```suggestion
The applications include expert elicitation, differential privacy, and inference in big data settings. 
```

Ryan will be focusing on algorithms for inference in big data settings for the next few years.

## Georgia Aitkenhead

> Georgia graduated from Cambridge university and She used to work for government.

Reviewer: ```suggestion
Georgia graduated from Cambridge University, after that she worked for the government.
```

> Now,She is currently working with Kirstie to design and build a citizen science platform(I think that is a great program). 

Reviewer: ```suggestion
Now, she is currently working with Kirstie to design and build a citizen science platform, which I think is a great program. 
```

This will be used to investigate sensory processing and navigating environments, but it can also be adapted to answer other research questions. 
She is co-designing an ethical framework to go along with the platform, based on methods of participatory science. 
Her research interests also include open source working and building a fine-grained consent model, as well as the psychology of interactions with technology, for example how to share data online.

> Georgia told me that she is really enjoying her research.

Reviewer: ```suggestion
Georgia told me that she is really enjoying her research, 
```

> Because it’s amazing to have this opportunity to immerse herself in the Turing’s collective expertise. 

Reviewer: ```suggestion
, because it’s amazing to have this opportunity to immerse herself in the Turing’s collective expertise. 
```

There are lots of avenues of growth which are possible from for current work, including developing skills in open source, programming languages, and the philosophy of technology/ethics. 

> She also told me some impressive experiences in Turing! (It is also impressive for me to make friend with Geogia!)

Reviewer: ```suggestion
She also told me some impressive experiences in Turing! (It is also impressive making friends with Georgia!)
```

Being a friend should begin with the right spell of her name.

> In the field of autism research, there has been a shift in the way that autism is spoken about and analysed and understood. 

Reviewer: ```suggestion
In the field of autism research, there has been a shift in the way that autism is spoken about and analyzed and understood. 

```

One example of this is a movement away from the trope of ‘defect’ to one of autistic pride, accomplishment, and the value of neurodiversity. 

## Chen Li

> Chen Li is a PhD student form Cambridge unversity and She is work for health program in the Alan Turing insititue.

Reviewer: ```suggestion
Chen Li is a Ph.D. student from Cambridge University and She is work for health program in the Alan Turing Institute.
```

Her major is Biology.

> She's interst is characterisation of Genomic ageing and the links to clinical phenotypes.

Reviewer: ```suggestion
Her interest is the characterization of Genomic aging and the links to clinical phenotypes.
```

Now,She is working on characterizing gene-specific effects of LDL cholesterol on type 2 diabetes using metabolomics.

> She have been take part in an azmazing research about a genetic variant protects against obesity, type 2 diabetes.

Reviewer: ```suggestion
She has been taking part in amazing research about a genetic variant protects against obesity, type 2 diabetes.
```

This gene is called the MC4R.It could protect people from being overweight.
MC4R could act as a "switch" in the brain that tells us not to eat after a meal.
Meanwhile, mutations in the MC4R gene, which is associated with a higher risk of obesity, can cause MC4R to stop working.
The research has been watched by drug companies who believe the genetic variation could help people achieve or maintain weight loss.

> Chen said she love working at Turing.She is very excited to get access to a lot of latest knowledge of artificial intelligence here.

Reviewer: ```suggestion
Chen said she loves working at Turing. She is very excited to get access to a lot of the latest knowledge of artificial intelligence here.
```
